docs/12.EXT-REFERENCE-MODULE.md

```diff
@@ -53,8 +54,12 @@ jerryx_module_resolve (const jerry_char_t *name,
 **Summary**
 
 The resolver for JerryScript modules. A pointer to this function can be passed in the second parameter to
-`jerryx_module_resolve` to search for the module among the JerryScript modules built into the binary. This function is
-available only if the preprocessor directive `JERRYX_NATIVE_MODULES_SUPPORTED` is defined.
+`jerryx_module_resolve()` to search for the module among the JerryScript modules built into the binary.
+
+**Note**: If compiler support for automatic loading of native modules is not available, indicated by the fact that the
+preprocessor directive `JERRYX_NATIVE_MODULES_SUPPORTED` is either absent or set to `0`, you must define an array of
```

Comment:
Here the zero check explicitly mentioned.

Reply:
I guess IRL it would not be defined. Only for the purposes of testing will it ever be defined but zero. So, should I leave "absent" and remove "set to &#96;0&#96;"?

Reply:
Probably yes. Although I like `#if defined M && M` In JerryScript we just test the existence of macros. Hence their value (1, 0, whatever) is irrelevant. Perhaps just remove all explicit number mentionings.

Reply:
OK, you're right. Maybe we should go one step further and not publish such feature macros at all. The only thing we need to document is that "if you know your compiler doesn't support linker section notation, (and if you know your compiler/C library doesn't support constructors), then call `JERRYX_NATIVE_MODULES_INIT()`" (still to be properly named).

Reply:
Not necessary. If somebody creates a generic library, he may want to support multiple module representations.